### PR TITLE
Update all.json

### PR DESCRIPTION
**Pull request by dubstard**
`"apps-bots.live",`
image

`"dapp.liveappio.com",`
image

`"web.cosmoioapp.com",`
image

`"networkbridgesvalidation.xyz",`
image

`"solanaspool.com",`
image

`"moonbeaim.network",`
image

`"validationapp.org",`
image

Pre dupe spam (spray n pray)
"24x7-bittreks.com",
"24x7-ledger.com",
"alienheroesworld.io",
"amodosecurityservices.org",
"app-onlinelottoclaim.us ",
"app.anchorappweb.com",
"app.anchorwebprotocol.com",
"app.cosmoioapp.com",
"app.multiversepad.net",
"apps-bots.live",
"assetsrestore.org",
"bonezai.com",
"bridge.tewrrra.money",
"browserhelper12.org",
"contentstack.store",
"cosmoioapp.com",
"dapp.liveappio.com",
"deal.smartokendeals.net",
"drinksonme.ng",
"elitefidelitytrade.com",
"event.eggroot.com",
"expaand.com",
"gate-site.xyz",
"giveaway-bananataskforceape.com",
"giveaway-blinmetaverse.com",
"giveaway-blueshift.com",
"giveaway-bscharactbit.com",
"giveaway-clonex.com",
"giveaway-cryptostake.com",
"giveaway-esg.com",
"giveaway-ezchain.com",
"giveaway-greenmeta.com",
"giveaway-helenafinancial.com",
"giveaway-konstrukt.com",
"giveaway-muesliswap.com",
"giveaway-paygo.com",
"giveaway-payzpayments.com",
"giveaway-pieme.com",
"giveaway-pleasurecoin.com",
"giveaway-safeukr.com",
"giveaway-senspark.com",
"giveaway-stepapp.com",
"giveaway-tifitoken.com",
"govgrantsupport.org",
"gpts3.gptscript.com",
"gptscript.com",
"greybull.digital",
"hedron.myappsio.com",
"liveappio.com",
"masterclass.binateclimited.com",
"meenaagro.com",
"mercadou.net",
"mints-nft.live",
"moneysandesh.com",
"moonbeaim.network",
"myappsio.com",
"navrangmontessori.org",
"networkbridgesvalidation.xyz",
"nft.binateclimited.com",
"p.rockeycouriers.com",
"puryrealgold.com",
"rbi-in.org",
"rockeycouriers.com",
"satgurun.net",
"seycee.com",
"solanalink.live",
"solanaspool.com",
"ssvnetwork.guard.ac",
"suncrossprinting.com",
"syncapp.assetsrestore.org",
"techgravity.xyz",
"tewrrra.money",
"trustpad-ido.com",
"uaarmy.com",
"uniswap.myappsio.com",
"validationapp.org",
"validereweb.com",
"web.cosmoioapp.com",

### CODE REVIEW

**Review by dubstard — COMMENTED** [review]
trim redundant space
"app-onlinelottoclaim.us ", ->"app-onlinelottoclaim.us",

**Review by dubstard — COMMENTED** [review]
Remove duplicates 
   -   "app.anchorwebprotocol.com",
   -   "assetsrestore.org",

File: all.json [hunk 1]
```diff
@@ -11,6 +11,82 @@
 		"162.222.226.37",
 		"1776coin.org",
 		"1chaincrypto.com",
+		"24x7-bittreks.com",
+		"24x7-ledger.com",
+		"alienheroesworld.io",
+		"amodosecurityservices.org",
+		"app-onlinelottoclaim.us",
+		"app.anchorappweb.com",
+		"app.cosmoioapp.com",
```
Comment (jacogr) [inline all.json:20]: Appears twice - `app` & `web` with no top-level blocks.

File: all.json [hunk 2]
```diff
@@ -11,6 +11,82 @@
...
+		"meenaagro.com",
+		"mercadou.net",
+		"mints-nft.live",
+		"moneysandesh.com",
+		"moonbeaim.network",
+		"myappsio.com",
+		"navrangmontessori.org",
+		"networkbridgesvalidation.xyz",
+		"nft.binateclimited.com",
+		"p.rockeycouriers.com",
```
Comment (jacogr) [inline all.json:71]: This just seems like a login page?

File: all.json [hunk 3]
```diff
@@ -11,6 +11,82 @@
...
+		"masterclass.binateclimited.com",
+		"meenaagro.com",
+		"mercadou.net",
+		"mints-nft.live",
+		"moneysandesh.com",
+		"moonbeaim.network",
+		"myappsio.com",
+		"navrangmontessori.org",
+		"networkbridgesvalidation.xyz",
+		"nft.binateclimited.com",
```
Comment (jacogr) [inline all.json:70]: Don't see anything here at this point in time...

File: all.json [hunk 4]
```diff
@@ -11,6 +11,82 @@
...
+		"p.rockeycouriers.com",
+		"puryrealgold.com",
```
Comment (jacogr) [inline all.json:72]: Don't see this having anything crypto related.

File: all.json [hunk 5]
```diff
@@ -11,6 +11,82 @@
...
+		"rbi-in.org",
+		"rockeycouriers.com",
```
Comment (jacogr) [inline all.json:74]: As above for the `p.` subdomain.

File: all.json [hunk 6]
```diff
@@ -11,6 +11,82 @@
...
+		"mints-nft.live",
+		"moneysandesh.com",
+		"moonbeaim.network",
+		"myappsio.com",
+		"navrangmontessori.org",
+		"networkbridgesvalidation.xyz",
+		"nft.binateclimited.com",
+		"p.rockeycouriers.com",
+		"puryrealgold.com",
+		"rbi-in.org",
```
Comment (jacogr) [inline all.json:73]: Unused?

File: all.json [hunk 7]
```diff
@@ -11,6 +11,82 @@
 		"162.222.226.37",
 		"1776coin.org",
 		"1chaincrypto.com",
+		"24x7-bittreks.com",
+		"24x7-ledger.com",
+		"alienheroesworld.io",
```
Comment (jacogr) [inline all.json:16]: Not sure i see the "trying to scam" here, it does look like an NFT?

**Review by dubstard — COMMENTED** [review]
add TLD for 
"anchorappweb.com",
"anchorwebprotocol.com",

remove 
"puryrealgold.com", (not crypto related, still scam)
"p.rockeycouriers.com", (not crypto related, still scam)

File: all.json (hunk 6 above)
Comment (dubstard) [reply all.json:73]: `nft.binateclimited.com` has nothing to do with `binance.com` and is an unborn scam.

File: all.json (hunk 5 above)
Comment (dubstard) [reply all.json:74]: removed this one

File: all.json (hunk 4 above)
Comment (dubstard) [reply all.json]: resolved

**Review by dubstard — COMMENTED** [review]
removed 		
"anchorappweb.com",
"anchorwebprotocol.com",

and subdomains related to them

File: all.json (hunk 7 above)
Comment (dubstard) [reply all.json:16]: "24x7-bittreks.com", = unborn bittrex.com scam (100% confidence)
"24x7-ledger.com", =  unborn ledger.com scam (100% confidence)

File: all.json (hunk 1 above)
Comment (dubstard) [reply all.json]: added the top levels. 
they were already there
removed subdomains and redundant TLDs

**Review by dubstard — COMMENTED** [review]
remove "masterclass.binateclimited.com",
(got mixed up with Binance)

**Review by dubstard — COMMENTED** [review]
remove 		
"navrangmontessori.org",

File: all.json (hunk 3 above)
Comment (dubstard) [reply all.json:66]: removing all but `networkbridgesvalidation.xyz`

**Comment by dubstard** [comment]
✅Ready for review (after numerous tweaks, thanks to Jaco)

**Review by dubstard — COMMENTED** [review]
remove 		"drinksonme.ng", (appears empty, on bad infra though)

**Review by dubstard — COMMENTED** [review]
remove 		
"event.eggroot.com", (blank)
"expaand.com", (not necessarily a scam/ phishing)

**Comment by dubstard** [comment]
✅Ready for review (after numerous tweaks, thanks to Jaco)

**Review by ical10 — APPROVED** [review]

**Comment by dubstard** [comment]
Thanks, Spray and pray complete 🚀

**Pull request merged**